Tests/scripts/validate_files.py

```diff
@@ -69,12 +70,38 @@ def is_py_script_or_integration(file_path):
 
             return True
 
+        if re.match(INTEGRATION_YML_REGEX, file_path, re.IGNORECASE):
+            if file_yml.get('type', 'javascript') != 'python':
```

Comment:
the default for `javascript` is wrong, it can also be `powershell`, just make it empty string